administrator/language/en-GB/tpl_atum.ini

```
@@ -9,15 +9,15 @@ TPL_ATUM_ALTTEXT_SITE_LOGO_LABEL="Alt Text Site Logo"
 TPL_ATUM_ALTTEXT_SMALL_LOGO_LABEL="Alt Text Small Site Logo"
 TPL_ATUM_BACKEND_LOGIN="Joomla Administrator Login"
 TPL_ATUM_BACK_TO_CONTROL_PANEL="Back to Dashboard"
-TPL_ATUM_COLORS_HUE="Choose your hue value for the dark template colour"
-TPL_ATUM_COLORS_SETTINGS_BG_DARK_LABEL="Dark Background"
+TPL_ATUM_COLORS_HUE="Choose your hue value for the primary template colour"
```

```suggestion
TPL_ATUM_COLORS_HUE="Choose the hue value for the primary template colour"
```

I think that reads better